src/Microsoft.AspNet.OData.Shared/Formatter/Serialization/ODataResourceSerializer.cs

```diff
@@ -275,7 +278,49 @@ private ResourceContext GetResourceContext(object graph, ODataSerializerContext
             }
         }
 
-        private async Task WriteDeltaComplexPropertiesAsync(SelectExpandNode selectExpandNode,
+        internal void WriteDeltaNavigationProperties(SelectExpandNode selectExpandNode, ResourceContext resourceContext, ODataWriter writer)
+        {
+            Contract.Assert(resourceContext != null);
```

Comment:
Like we agreed, lets pass the second argument of `Contract.Assert` method since that's what is written to the output window when debugging